secret_number.js

```diff
@@ -1,4 +1,25 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+
+  // Create Random Number
+  let secretNumber = function() {
```

Comment:
Instead of using a function to generate a random number, you could just generate it and set it to `generatedNumber`. Only the `storedNumber` function is really needed for external use.